docs/openapi.yaml

```diff
@@ -0,0 +1,826 @@
+          to do the conversion, therefore print parameter such as margins and
+          paper size are also accepted as optional parameters.
+        content:
+          multipart/form-data:
+            schema:
+              $ref: '#/components/schemas/URLConvertRequestBody'
+            examples: { }
+      responses:
+        '200':
+          $ref: '#/components/responses/SuccessfulPDF'
+        '400':
+          description: Bad Request
+
+  /forms/chromium/convert/html:
+    post:
+      tags:
+        - chromium
+      summary: Convert a given HTML file to PDF
+      description: >-
+        Send an HTML file called `index.html` as a multipart form request, and
+        get the resulting PDF file. You can optionally include `header.html` and
+        `footer.html` files as part of the request as well.
+
+
+        An example `index.html` file can be as follows:
+
+        ```html
+
+        <!doctype html>
+
+        <html lang="en">
+          <head>
+            <meta charset="utf-8">
+            <title>My PDF</title>
+          </head>
+          <body>
+            <h1>Hello world!</h1>
+          </body>
+        </html>
+
+        ```
+
+
+        You may also add a header and/or a footer in the resulting PDF  by
+        sending a file named `header.html` and `footer.html` respectively. Both
+        the header and the footer files has to be a complete HTML document. An
+        example `footer.html` can be as follows:
+
+        ```html
+
+        <html>
+            <head>
+                <style>
+                    body {
+                        font-size: 8rem;
+                        margin: 4rem auto;
+                    }
+                </style>
+            </head>
+            <body>
+                <p>
+                    <span class="pageNumber"></span> of <span class="totalPages"></span>
+                </p>
+            </body>
+        </html>
+
+        ```
+
+
+        The following classes will allow you to inject printing values in your
+        document:
+
+        - `date`: Formatted print date.
+
+        - `title`: Document title.
+
+        - `pageNumber`: Current page number.
+
+        - `totalPage`: Total pages in the document.
+
+
+        There are some limitations with header and footer files:
+
+        - JavaScript is not executed.
+
+        - External resources are not loaded.
+
+        - The CSS properties are independant of the ones used in the
+        `index.html` file.
+
+        - `footer.html` CSS properties override the ones from `header.html`.
+
+        - Only fonts installed in the Docker image are loaded (see the fonts
+        section)
+
+        - Images only work using a `base64` encoded source, e.g. `<img
+        src="data:image/png;base64, iVBORw0K... />`
+
+        - `background-color` and `color` CSS properties require an additional
+        `-webkit-print-color-adjust: exact` CSS property in order to work.
```

Comment:
I'm note sure we should specify everything as most of the details are available in the documentation anyway 🤔

Reply:
I totally agree with you. I just kept these lines, because they were part of the V6 API Spec, but they are redundant with the official documentation and it is cumbersome to maintain this information in both places.

Reply:
I added externalDocs to every api endpoint, so users of this api do not have to scroll up to the **tags** part if the api spec (where the exterenDocs were added earlier). In addition to that I deleted these details from the specification.